src/index.html

```diff
@@ -7,6 +7,39 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day_fri calendar--month-length_29">
+      <div class="calendar__box-1 box"></div>
+      <div class="calendar__box-2 box"></div>
+      <div class="calendar__box-3 box"></div>
```

Comment:
you should use styles instead of hardcoding
```suggestion
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
```